src/image.c

```diff
@@ -38,51 +38,45 @@ static WOLFTPM2_DEV wolftpm_dev;
 #ifdef WOLFBOOT_SIGN_ED25519
 #include <wolfssl/wolfcrypt/ed25519.h>
 
-static int wolfBoot_verify_signature(uint8_t *hash, uint8_t *sig)
+static void wolfBoot_verify_signature(struct wolfBoot_image *img, uint8_t *sig)
 {
     int ret, res;
     ed25519_key ed;
     ret = wc_ed25519_init(&ed);
     if (ret < 0) {
         /* Failed to initialize key */
-        return -1;
+        return;
     }
     ret = wc_ed25519_import_public(KEY_BUFFER, KEY_LEN, &ed);
     if (ret < 0) {
         /* Failed to import ed25519 key */
-        return -1;
-    }
-    ret = wc_ed25519_verify_msg(sig, IMAGE_SIGNATURE_SIZE, hash, WOLFBOOT_SHA_DIGEST_SIZE, &res, &ed);
-    if ((ret < 0) || (res == 0)) {
-        return -1;
+        return;
     }
-    return 0;
+    VERIFY_FN(img, &res, wc_ed25519_verify_msg, sig, IMAGE_SIGNATURE_SIZE,
+            img->sha_hash, WOLFBOOT_SHA_DIGEST_SIZE, &res, &ed);
 }
 
 #endif /* WOLFBOOT_SIGN_ED25519 */
 
 #ifdef WOLFBOOT_SIGN_ED448
 #include <wolfssl/wolfcrypt/ed448.h>
 
-static int wolfBoot_verify_signature(uint8_t *hash, uint8_t *sig)
+static void wolfBoot_verify_signature(struct wolfBoot_image *img, uint8_t *sig)
```

Comment:
Please add code comment indicating the verify success is returning in `wolfBoot_image`.

Reply:
:+1:  (added in the caller since this function has many prototypes)